owncloudApp/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -312,7 +312,7 @@ public boolean onTouch(View v, MotionEvent event) {
                         action.putExtra(Intent.EXTRA_ALLOW_MULTIPLE, true);
                         getActivity().startActivityForResult(
                                 Intent.createChooser(action, getString(R.string.upload_chooser_title)),
-                                FileDisplayActivity.REQUEST_CODE__SELECT_CONTENT_FROM_APPS
+                                FileDisplayActivity.Companion.getREQUEST_CODE__SELECT_CONTENT_FROM_APPS()
```

Comment:
the constant is strange, isn't it? some parts are in lowercase, others in uppercase...

Reply:
The name of the variable is `REQUEST_CODE__SELECT_CONTENT_FROM_APPS`, as you can see [here](https://github.com/owncloud/android/blob/4c444efd4ce9ba0639fe5e95b77ec69635f13615/owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java#L139), but since we are calling it from java this is how Android Studio solved it when transforming https://github.com/owncloud/android/blob/4c444efd4ce9ba0639fe5e95b77ec69635f13615/owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java to Kotlin.